e2e/msp-generic-tests.ts

```diff
@@ -0,0 +1,77 @@
+import { browser } from 'protractor';
+import { BaseMSPTestPage } from './msp.po';
+
+const page = new BaseMSPTestPage();
```

Comment:
```suggestion

```

Remove this entirely. Generic tests should not have any instance in the file (i.e. you shouldn't call 'new Page()' here).

Instead:

1. Remove the `page` constant from this file
2. Change "testLoadPage" so that it takes a "BaseMSPTestPage" as it's second parameter
3. When you call it in your tests, call it like:

```
testPageLoad(REVIEW_PAGE_URL, new ReviewPage());
```

Note - you MUST call it with `new ReviewPage()` in tests instead of passing a variable defined in beforeEach.  Why? Because it looks like the beforeEach() cannot detect our generic tests, so we have to pass it the instance itself.

Another option to look at is exporting "describe()" blocks instead of exporting "it()" blocks.  Then you'd basically do:

```typescript
export function testGeneric(pageURL: string, page: BaseMSPTestPage){

   return describe('GENERIC TESTS' => {
    // would have it's own beforeEach 
    // would call all the generic tests you've written so far
   }

}
````

After you remove the local instance and have that working, please take a look at exporting describe blocks.  I think it'll be a lot simpler, and it looks like exporting describe() is more supported than exporting it().